### ISSUE TITLE
Channel size estimate dtype error

### ISSUE THREAD

**Issue by kkappler** [kujaku11/mth5#60]
Inside master_station_run_channel.py around line 1134 on branch make_mth5_51 the code was taking the difference between time_period.end and time_period.start.

This fails because they are both strings.  Below is a snippet that fixes the issue.

REPLACE
```
estimate_size = (
                            channel_metadata.time_period.end - channel_metadata.time_period.start) * channel_metadata.sample_rate

```
which acutally should return a tuple WITH
```
if channel_metadata.time_period.start != channel_metadata.time_period.end:
  if channel_metadata.sample_rate > 0:
      end_time = UTCDateTime(channel_metadata.time_period.end)
      start_time = UTCDateTime(channel_metadata.time_period.start)
      delta_time_seconds = end_time - start_time
      n_samples_ish = delta_time_seconds * \
                      channel_metadata.sample_rate
      estimate_size = (int(n_samples_ish),)
```

N.B. A better solution is to add a "duration" method to channel_metadata, so that it returns the number of seconds, i.e.:

```
if channel_metadata.time_period.start != channel_metadata.time_period.end:
  if channel_metadata.sample_rate > 0:
      delta_time_seconds = channel_metadata.time_period.duration
      n_samples_ish = delta_time_seconds * \
                      channel_metadata.sample_rate
      estimate_size = (int(n_samples_ish),)

```
@timronan I think you were going to add a temporary fix on the make_mth5_51 branch

**Comment by timronan** [comment]
@kkappler this seems to have been merged by PR #55 and is under commit https://github.com/kujaku11/mth5/pull/55/commits/8c0ed15e787ae2d28cfc12882d064452e3d2973e. 

PR #59:
```
               if channel_metadata is not None:
                    # can estimate a size, this will help with allocating
                    # and set the chunk size to a realistic value
                    if (
                        channel_metadata.time_period.start
                        != channel_metadata.time_period.end
                    ):
                        if channel_metadata.sample_rate > 0:
                            estimate_size = (
                                int(
                                    (
                                        channel_metadata.time_period._end_dt
                                        - channel_metadata.time_period._start_dt
                                    )
                                    * channel_metadata.sample_rate
                                ),
                            )
                else:
```
Seems like it overwrote this logic. I think it would be easiest to just  take the logic in commit c0ed15e787ae2d28cfc12882d064452e3d2973e. 

```
                       end_time = UTCDateTime(channel_metadata.time_period.end)
                        start_time = UTCDateTime(channel_metadata.time_period.start)
                        delta_time_seconds = end_time - start_time
                        n_samples_ish = delta_time_seconds * channel_metadata.sample_rate
                        estimate_size = (int(n_samples_ish),)
```
and make a small pr for it so this issue can quickly be navigated. What does everyone think?

**Comment by kujaku11** [comment]
I think this has been fixed.
